lib/node/runner/checker_ballot_transaction.go

```diff
@@ -225,17 +225,10 @@ func ValidateTx(st *storage.LevelDBBackend, config common.Config, tx transaction
 		return
 	}
 
-	// get the balance at sequenceID
-	var bac block.BlockAccountSequenceID
-	bac, err = block.GetBlockAccountSequenceID(st, tx.B.Source, tx.B.SequenceID)
-	if err != nil {
-		return
-	}
-
 	totalAmount := tx.TotalAmount(true)
 
 	// check, have enough balance at sequenceID
-	if bac.Balance < totalAmount {
```

Comment:
> @kfangw You pointed, this block does not used in SEBAK, but I think this will be useful to check the change history of `BlockAccount`. How do you think?

Although that purpose, this should be changed like that I changed.

Reply:
This change can be merged without debating, I think.
Hence, make another PR for it.
https://github.com/bosnet/sebak/pull/775